shotover/src/hot_reload/server.rs

```diff
@@ -12,21 +14,17 @@ use uds::tokio::{UnixSeqpacketConn, UnixSeqpacketListener};
 pub struct SourceHandle {
     pub name: String,
     pub sender: mpsc::UnboundedSender<HotReloadListenerRequest>,
+    pub gradual_shutdown_tx: mpsc::UnboundedSender<GradualShutdownRequest>,
```

Comment:
should these channels be oneshot?

Reply:
Theres also some existing channels that could be oneshots as well, I've added an item to https://github.com/shotover/shotover-proxy/issues/1915

Since you've given approval, I'll merge the PR as is, and that item can be addressed in the future.